modules/shortcodes/polldaddy.php

```diff
@@ -573,7 +573,7 @@ function polldaddy_link( $content ) {
 	add_filter( 'the_content_rss', 'polldaddy_link', 1 );
 
 	/** This filter is documented in modules/shortcodes/youtube.php */
-	if ( apply_filters( 'jetpack_comments_allow_oembed', get_option( 'embed_autourls' ) ) ) {
+	if ( apply_filters( 'jetpack_comments_allow_oembed', true ) ) {
```

Comment:
Thank you for the explanation, @mdawaffe 

I didn't find many references to this option online, that's why I thought that it could be safe enough to remove the query completely.

In any case, If we want to maintain backward compatibility, we could use the same pattern as for the other options, and set a default value. What do you think?